### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this project I have:

- added linter setup and fix all linter errors
- added web pack plugins and connected files
- added crud operations on todo tasks

To run this project in your system please consider the following practices:

- [x] clone this GitHub repository `git clone https://github.com/KUBAHO3/To-Do-List` and navigate to the add-remove branch `cd To-Do-List`, `git checkout add-remove`
- [x] install all node packages by `npm install`
- [x] run the following command in your terminal to build the system on your dev `npm run build`
- [x] Navigate to the `base_url/dist/`

### CODE REVIEW

File: src/modules/operations.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    });
+  }
+
+  
+  static removeTask(index) {
+    if (index > -1) {
+      const newData = this.getAllTasks();
+      newData.splice(index, 1);
+      for (let i = index; i < this.getAllTasks().length - 1; i += 1) {
+        newData[i].index = String(i, 10);
```
Comment (TedLivist) [inline src/modules/operations.js]: - [ ] When a task is deleted the remaining tasks, downward to the last have their `index` values updated to strings. Kindly note that the `index` value should always be a number (integer), not a string. Kindly fix that

  ```
  newData[i].index = i;
  ```

  image

File: src/modules/operations.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+export  class Operations {
+  static createTask() {
+    const inputData = document.getElementById('inputText').value;
+    const saveData = document.getElementById('formText');
+    const storage = this.getAllTasks().length;
+    const task = [{ description: inputData, completed: false, index: storage }];
+    this.saveTask(task);
+    this.showTask();
+    saveData.reset();
+  }
```
Comment (TedLivist) [inline src/modules/operations.js]: - [ ] When a task is added its value should be equal to its position on the array, starting from count 1. That means the first task should have an `index` of 1, and the 13th task should have an `index` of 13. Your index value starts at 0, kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @KUBAHO3,

Good job so far!
✔️ Editing of tasks work as expected
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by adel-gu — APPROVED** [review]
# Status: APPROVED 🟢 

**Hi @KUBAHO3 👋🏼** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- The tasks indexes are numbers ✔️ 
- The Indexes start counting from 1 ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**